config/environments/local/local.node.config.toml

```diff
@@ -72,6 +72,7 @@ LastBatchVirtualizationTimeMaxWaitPeriod = "300s"
 WaitBlocksToUpdateGER = 10
 WaitBlocksToConsiderGerFinal = 10
 ElapsedTimeToCloseBatchWithoutTxsDueToNewGER = "60s"
+MinTimeToCloseBatch = "60s"
```

Comment:
I think we also need to include this field into public.node.config.toml

Reply:
@ARR552 I didn't add it because there is no `[Sequencer]` section in the `public.node.config.toml` file. I think it's because at this moment the public node is meant to rely on other nodes for sequencing batches. Does it make sense to you?